devops/infrastructure/github-action-playbook.yml

```diff
@@ -16,6 +16,11 @@
           name: common
           tasks_from: run-setup-build
 
+      - name: Install subkey
+        include_role:
+          name: admin
+          tasks_from: main
+
```

Comment:
Also tried running subkey install in setup.sh https://github.com/Joystream/joystream/commit/52f0d99eafae6cb49c72c75454d3ca1d3a22c783